src/pages/timecard/Timecard.jsx

```diff
@@ -20,11 +20,26 @@ const Timecard = () => {
     document.title = generateDocumentTitle('Timecard');
   });
 
+  const sortSummaryErrors = () => {
```

Comment:
I guess this is an example of my previous comment, looks like we're duplicating the same kind of logic.

Reply:
As above, I've now moved this function to a separate class